index.html

```diff
@@ -0,0 +1,677 @@
+                  class="logo-icons2"
+                  alt="union"
+              /></a>
+            </li>
+          </ul>
+
+          <div>
+            <button class="get_my_resume">Get my resume</button>
+          </div>
+        </div>
+
+        <div class="languages_frames">
+          <h4>Languages</h4>
+          <div class="frame">
+            <img src="images_mobile/Union.jpg" alt="union" />
+          </div>
+
+          <ul class="languages">
+            <li>
+              <img src="./images_mobile/elipse_js.jpg" alt="js" />
+              <strong>JavaScript</strong>
+            </li>
+
+            <li>
+              <img src="./images_mobile/elipse_html.jpg" alt="html" />
+              <strong>HTML</strong>
+            </li>
+
+            <li>
+              <img src="./images_mobile/elipse_css.jpg" alt="css" />
+              <strong>CSS</strong>
+            </li>
+          </ul>
+
+          <h4>Frameworks</h4>
+
+          <div class="frame">
+            <img src="images_mobile/icon-chevron.jpg" alt="icon" />
+          </div>
+
+          <hr />
+
+          <h4>Skills</h4>
+          <div class="frame">
+            <img src="images_mobile/icon-chevron.jpg" alt="icon" />
+          </div>
+        </div>
+      </section>
+      <br />
+      <br />
+      <section class="contact_form_container">
+        <h1 class="contactme-title">Contact me</h1>
+        <div class="contactme-decoration"></div>
+
+        <p class="contactme-subtitle1">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <footer>
+          <form
+            class="contact_last_session"
+            action="https://formspree.io/f/xqkngjdg"
+            method="POST"
+          >
+            <ul class="form_contact">
+              <li>
+                <input
+                  type="text"
+                  class="text1"
+                  name="user_name"
+                  maxlength="30"
+                  placeholder="Yeremias"
+                  pattern="[a-z]{1,30}"
+                  required
+                  title="Username should only contain lowercase letters. e.g John"
+                />
+              </li>
+              <li>
+                <input
+                  type="email"
+                  class="text1"
+                  size="32"
+                  name="contact_email"
+                  maxlength="32"
+                  placeholder="natajayanj14@gmail.com"
+                  required
+                  pattern=".+@\.com"
+                  title="Please Enter Valide Email Id"
+                />
+              </li>
+              <li>
+                <input
+                  class="msg"
+                  maxlength="500"
+                  name="user_message"
+                  placeholder="Write your message here"
+                />
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the message field is a `textarea` instead of an input.